### ISSUE TITLE
improvement: request params fetch mode optimization

### ISSUE THREAD

**Issue by hehehai** [mswjs/msw#47]
### background:
pr: #46 , fixs: #43 
https://github.com/open-draft/msw/blob/325cf4bbd7247bbb4a4cb7ca5c48b7ace1844f27/src/handleIncomingRequest.ts#L31-L38

### optimize:
```ts
const params = relevantRequestHandler.mask
	? matchPath(req.url, { path: relevantRequestHandler.mask }).params
    : {}

const requestWithParams = {
    ...req,
    params,
}
```

### why:
it will be clearer to take out the `params` directly from the `match`. We just used the `params` of `matchPath`

**Comment by kettanaito** [comment]
Sounds like a good idea 👍 Go for it.

**Comment by kettanaito** [comment]
Closed by #54. Will be released in the next release. Thanks!

Feel free to share your ideas and suggestions by opening an issue or a pull request. If you feel like it, there's a great need for the support of serialization/deserialization multiple headers with the same name (#21).
